docs/sdd.md

```diff
@@ -496,3 +496,154 @@ sequenceDiagram
         end
     end
 ```
+
+### Data model CockroachDB
+
+| Value (left) | Value (right) | Meaning                       |
+| ------------ | ------------- | ----------------------------- |
+| \|o          | o\|           | Zero or one                   |
+| \|\|         | \|\|          | Exactly one                   |
+| }o           | o{            | Zero or more (no upper limit) |
+| }\|          | \|{           | One or more (no upper limit)  |
+
+```mermaid
+erDiagram
+    flight_plan {
+        uuid flight_plan_id PK
+        uuid pilot_id FK
+        uuid vehicle_id FK
+        integer flight_distance
+        text weather_conditions
+        uuid departure_vertipad_id FK
+        uuid destination_vertipad_id FK
+        timestamp scheduled_departure
+        timestamp scheduled_arrival
+        timestamp actual_departure "Optional"
+        timestamp actual_arrival "Optional"
+        timestamp flight_release_approval "Optional"
+        timestamp flight_plan_submitted "Optional"
+        uuid approved_by FK "Optional"
+        json cargo_weight_g "Optional"
+        text flight_status "Optional"
+        text flight_priority "Optional"
+        timestamp created_at "Default NOW"
+        timestamp updated_at "Default NOW"
+        timestamp archived_at "Default NULL"
+    }
+    vertiport {
+        uuid vertiport_id PK
+        text description
+        float longitude
+        float latitude
+        text schedule "Optional"
+        timestamp created_at "Default NOW"
+        timestamp updated_at "Default NOW"
+        timestamp deleted_at "Default NULL"
+    }
+    vertipad {
+        uuid vertipad_id PK
+        text description
+        uuid vertiport_id FK
+        float longitude
+        float latitude
+        text schedule "Optional"
+        bool enabled "Default true"
+        bool occupied "Default false"
+        timestamp created_at "Default NOW"
+        timestamp updated_at "Default NOW"
+        timestamp deleted_at "Default NULL"
+    }
+    user {
```

Comment:
The idea is that a `contact` doesn't necisserily need to be someone that has a log in to the system.
A `user` is an actual system user that has log in information.
In addition, a user can have contact information, so I think it would make sense to have a user_contact table linking the 2

Reply:
:+1: